test/client.go

```diff
@@ -52,14 +57,14 @@ func run() error {
 		}
 		return err
 	}
-	stream, err := ipCam.Stream(ctx)
-	if err != nil {
-		return err
-	}
-	_, _, err = stream.Next(ctx)
-	if err != nil {
-		return err
+
+	for range maxRetries {
+		_, err = camera.DecodeImageFromCamera(ctx, "image/jpeg", nil, ipCam)
+		if err == nil {
```

Comment:
you know... sicne these are strings we could just implement YUV returns from viamrtsp right now

Reply:
meant to comment on line 62